dts/bindings/net/wireless/nordic,nrf-radio.yaml

```
@@ -0,0 +1,103 @@
+# Copyright (c) 2020 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+description: |
+    Nordic DFE antennas matrix description
```

```suggestion
    Nordic nRF family RADIO.
```

To match the other nRF bindings, since you've changed the compatible (and thanks for that).

---

Corrected, thanks for spotting that.